vmware_exporter/vmware_exporter.py

```diff
@@ -15,6 +15,7 @@
 import sys
 import traceback
 import pytz
+import logging
 
```

Comment:
I think it's more idiomatic to create a named logger rather than using the root one?

```
logger = logging.getLogger(__name__)
```

then `logger.info` instead of `logging.info` etc.

Reply:
Naming is hard, so I sticked to the root package for the application.
Someones might just prefer log over logger other than logging.